CollapseLauncher/Classes/Helper/Background/Loaders/StillImageLoader.cs

```diff
@@ -0,0 +1,223 @@
+        {
+            while (IsImageLoading)
+            {
+                await Task.Delay(250, token);
+            }
+
+            if (!IsImageDimm)
+            {
+                await ToggleImageVisibility(true);
+            }
+
+            IsImageDimm = true;
+        }
+
+        public async ValueTask UndimmAsync(CancellationToken token)
+        {
+            while (IsImageLoading)
+            {
+                await Task.Delay(250, token);
+            }
+
+            if (IsImageDimm)
+            {
+                await ToggleImageVisibility(false);
+            }
+
+            IsImageDimm = false;
+        }
+
+        private async ValueTask ToggleImageVisibility(bool hideImage, bool completeInvisible = false)
+        {
+            TimeSpan duration = TimeSpan.FromSeconds(hideImage
+                ? BackgroundMediaUtility.TransitionDuration
+                : BackgroundMediaUtility.TransitionDurationSlow);
+
+            if (completeInvisible)
+            {
+                await Task.WhenAll(
+                    ImageBackParentGrid.StartAnimation(
+                        duration,
+                        CurrentCompositor.CreateScalarKeyFrameAnimation("Opacity",
+                            hideImage ? completeInvisible ? 0f : 0.4f : 1f, hideImage ? 1f : 0f)
+                    )
+                );
+            }
+            else
+            {
+                await Task.WhenAll(
+                    AcrylicMask.StartAnimation(
+                        duration,
+                        CurrentCompositor.CreateScalarKeyFrameAnimation("Opacity",
+                            hideImage ? 1f : 0f, hideImage ? 0f : 1f)
+                    ),
+                    OverlayTitleBar.StartAnimation(
+                        duration,
+                        CurrentCompositor.CreateScalarKeyFrameAnimation("Opacity",
+                            hideImage ? 0f : 1f, hideImage ? 1f : 0f)
+                    )
+                );
+            }
+        }
+
+        public async ValueTask ShowAsync(CancellationToken token)
+        {
+            await ToggleImageVisibility(false, true);
+        }
+
+        public async ValueTask HideAsync(CancellationToken token)
+        {
+            await ToggleImageVisibility(true, true);
+        }
+
+        public void Mute()
+        {
+            LauncherConfig.SetAndSaveConfigValue("BackgroundAudioIsMute", true);
+        }
+
+        public void Unmute()
+        {
+            LauncherConfig.SetAndSaveConfigValue("BackgroundAudioIsMute", false);
+        }
+
+        public void SetVolume(double value)
+        {
+            LauncherConfig.SetAndSaveConfigValue("BackgroundAudioVolume", value);
+        }
+
+        public void WindowFocused()
+        {
+        }
+
+        public void WindowUnfocused()
+        {
+        }
+
+        public void Play()
+        {
+        }
+
+        public void Pause()
+        {
+        }
```

Comment:
This is required since those methods are members of ``IBackgroundMediaLoader`` interface. But since ``StillImageLoader`` doesn't have such functions, I left it empty.